examples/dti_prediction/model.py

```diff
@@ -0,0 +1,78 @@
+import pytorch_lightning as pl
+import torch
+from torch.nn import functional as F
+
+from kale.embed.deepdta import DeepDTAEncoder, MLPDecoder
+
+
+class LitDeepDTA(pl.LightningModule):
+    def __init__(self, num_drug_embeddings, drug_dim, drug_length, num_filters, drug_filter_length,
+                 num_target_embeddings, target_dim, target_length, target_filter_length, decoder_in_dim,
+                 decoder_hidden_dim, decoder_out_dim, dropout):
```

Comment:
lots of arguementns (13). Possible to be more compact by introducing some structure? e.g. tuples?